docs/source/rule-testing.rst

```diff
@@ -76,6 +76,84 @@ For example, to replace a time based field with ``last_hour``:
   }
 
 
+Validating Log Schemas
+~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases, there may be logs coming into StreamAlert that are of a known type, but you do not have a current need to apply rule
+logic to them. However, it is still advantageous to add schemas for these logs and *test* to ensure your new schemas are valid.
+
+This is possible through the addition of log schema(s) to ``conf/logs.json`` and creation of test record(s) in ``tests/integration/rules/``
+using known samples of logs (without actually adding a corresponding rule). Running the ``manage.py`` script with the ``validate-schemas``
+option will iterate over all json test files and attempt to match records within them to a defined log schema.
+
+To simply run schema validation on all test files:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas
+
+
+To run schema validation on a specific test file within ``tests/integration/rules/``:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas --test-files <test_rule_file.json>
+
+Or:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas --test-files <test_rule_file>
+
+
+Example of running schema validation on two valid test files:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas --test-files cloudtrail_critical_api_calls cloudtrail_put_bucket_acl.json
+
+This will produce output similar to the following::
+
+  cloudtrail_critical_api_calls
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - DeleteSubnet
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - DeleteVpc
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - UpdateTrail
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - StopLogging
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - DeleteDBCluster
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - StopConfigurationRecorder
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - DeleteFlowLogs
+         [Pass]  [log='cloudtrail:events']     validation  (s3): CloudTrail - Critical API - False Positive Case
+
+  cloudtrail_put_bucket_acl
+         [Pass]  [log='cloudwatch:events']     validation  (kinesis): CloudTrail - PutBucketAcl - True Positive
+         [Pass]  [log='cloudwatch:events']     validation  (kinesis): CloudTrail - PutBucketAcl - False Positive
+
+
+
+  StreamAlertCLI [INFO]: (10/10) Successful Tests
+  StreamAlertCLI [INFO]: Completed
+
+
+Example of running schema validation on a test file that contains one valid record and one invalid record:
```

Comment:
I think explaining what is actually in the output makes it more obvious to the user what is happening. Saying `Failed test` doesn't really explain what failed very well